.circleci/config.yml

```diff
@@ -1,9 +1,9 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7
```

Comment:
we don't want to pin these to specific versions just using 7 will use the latest version of 7.
Also there is hmpps@9 of the orb

Reply:
do we want to upgrade that to 9 then?